source/common/ssl/context_config_impl.cc

```diff
@@ -139,17 +169,21 @@ ServerContextConfigImpl::ServerContextConfigImpl(
         return ret;
       }()) {
   // TODO(PiotrSikora): Support multiple TLS certificates.
-  if (config.common_tls_context().tls_certificates().size() != 1) {
+  if (config.common_tls_context().tls_certificates().size() != 1 &&
+      config.common_tls_context().tls_certificate_sds_secret_configs().size() != 1) {
```

Comment:
We can have only 1 in total, so perhaps test `if (sum != 1)`.

Reply:
Done